src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -323,16 +267,52 @@ export function TransactionRow ({ transaction, setActionsVisibleId, actionsVisib
     fullNotes = isOffer ? ` Declined Offer: ${notes}` : ` Declined Request: ${notes}`
   } else fullNotes = notes
 
-  const disabledTransaction = disableActionedTransaction(transaction.id)
-  let highlightGreen, highlightRed, highlightNeutral
-  if (disabledTransaction) {
-    highlightGreen = increaseAction()
-    highlightRed = decreseAction()
-    highlightNeutral = statusQuoAction()
+  const [highlightGreen, setHighlightGreen] = useState(false)
+  const [highlightRed, setHighlightRed] = useState(false)
+  const [isDisabled, setIsDisabled] = useState(false)
+  const [isVisible, setIsVisible] = useState(true)
```

Comment:
Can we use this same trick in the transaction history page?  Such that the row disappears whenever we receive back a successful mutation?